app/controllers/api/v0/sources_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
       private
 
       def create_params
-        body_params.permit(:name, :source_type_id, :tenant_id)
+        body_params.permit(:name, :source_type_id, :tenant_id, :uid)
```

Comment:
Should we really be having the client give us a uid? Would it be better to generate one on the model?

Reply:
I was thinking there might be cases where we're using the native provider uuid but that doesn't apply to any of these providers yet so yeah i'll just generate it

Reply:
Does this line still need to be here? Or can this be removed? It looks like we'll overwrite anything that is passed in the params based on the `.merge!` above, right?

Reply:
Yup just forgot to delete it